search-setup/integration_tests/search.test.ts

```diff
@@ -58,9 +88,52 @@ describe('AI search index creation and data loading', () => {
 
     it('should populate indicator index with data', async () => {
       const url = `${urlPrefix}/docs${URL_SUFFIX}`;
-
       await expectIndexToBePopulated(url);
     });
+
+    it('should return indicators for readmissions', async () => {
+      const response = await searchIndicatorsRequest('readmissions');
```

Comment:
I'm a little confused by these tests. They seemed to be just testing the function defined above in the test file. What is this actually meant to be testing? This could be my lack of understanding around this area.

Reply:
ah nice 👍  - The clue was the integration_tests directory 😂